guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -170,3 +171,42 @@ This removes Pulp 2 RPMs, content in `/var/lib/pulp/content/`, the mongo databas
 ----
 # hammer capsule content synchronize --name ${{SmartProxy}} --skip-metadata-check true --async
 ----
+
+.Upgrading {SmartProxy} Servers Using Remote Execution in the {Project} web UI
+
+. Create a backup.
++
+* On a virtual machine, take a snapshot.
+* On a physical machine, create a backup.
++
+For information on backups, see {AdministeringDocURL}backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProjectVersionPrevious}_ guide.
+
+. Optional: If you made manual edits to DNS or DHCP configuration in the `/etc/zones.conf` or `/etc/dhcp/dhcpd.conf` files, back up the configuration files because the installer only supports one domain or subnet, and therefore restoring changes from these backups might be required.
+
+. Optional: If you made manual edits to DNS or DHCP configuration files and do not want to overwrite the changes, enter the following command:
++
+[options="nowrap"]
+----
+# {satellite-installer --foreman-proxy-dns-managed=false \
```

Comment:
```suggestion
# {satellite-installer} --foreman-proxy-dns-managed=false \
```
Do you build your docs before submitting the PR?

Reply:
Every time. Sometimes I build several times.

Reply:
I must have overlooked that one in the last build.